quetz/config.toml

```diff
@@ -0,0 +1,11 @@
+[github]
+# Register the app here: https://github.com/settings/applications/new
+client_id = "{}"
+client_secret = "{}"
+
+[sqlalchemy]
+database_url = "{}"
+
+[session]
+secret = "{}"
+https_only = {}
```

Comment:
Would it make sense to suffix the name of this file with `.template`?

Reply:
Agree!

It's planned to use traitlets to perform validation?
If not sure there equivalent of schema schema validation implemented in toml.

Reply:
Can't say I've heard of traitlets. I have a habit of trying to keep dependencies as light as possible, but I have no objections.

I imagine we could validate against a JSON schema.

Reply:
This is what I had in mind.
For a next PR?

Reply:
Fine by me.